app/components/sdg_management/local_targets/index_component.html.erb

```diff
@@ -0,0 +1,41 @@
+<%= header do %>
+  <%= link_to t("sdg_management.local_targets.index.new"),
+              new_sdg_management_local_target_path,
+              class: "new-link" %>
+<% end %>
+
+<%= render SDGManagement::SubnavigationComponent.new(current: :local_targets) do %>
+  <table>
+    <thead>
+      <tr>
+        <td></td>
+        <th id="local-target-title" scope="col"><%= attribute_name(:title) %></th>
+        <th id="local-target-actions" scope="col"><%= t("admin.actions.actions") %></th>
+      </tr>
+    </thead>
+
+    <tbody>
+      <% local_targets.group_by(&:target).map do |target, local_targets| %>
+        <tr class="target-header">
+          <th id="<%= header_id(target) %>" colspan="3" scope="colgroup">
+            <%= target.code %> <%= target.title %>
```

Comment:
Maybe `code_and_title(target)`? :thinking:

Reply:
This method does not exist at `sdg_management/local_targets/index_component.rb` maybe you saw the one I added to the FormComponent. Do you think its a good idea to add it here too?

Reply:
You're absolutely right; I misread. It's fine like this :ok_hand:.